examples/reference-entity-model/entities/Product.entity.json

```diff
@@ -14,8 +14,18 @@
         "productName": {
           "datatype": "string",
           "collation": "http://marklogic.com/collation/codepoint"
+        },
+        "category": {
```

Comment:
How would be tell it apart as a Concept from the "basic" properties? ( those that are integer, boolean, string, etc)
For properties related to strucuted properties we do not put datatype:string
`"address": {
          "$ref": "#/definitions/Address"
        },`
I think we should follow the same pattern and do not say the type is string when the type would be "concept".
Maybe we can add "$ref": "#/concepts/Category" ? Or something along thoes lines? A new $concept attribute with the base http://www.example.com/Category?

Overall, I like how you thought about adding the Concept as part of a property to have a smooth flow.